src/bluehawk/parser/lexer/makeLexer.ts

```diff
@@ -8,23 +8,25 @@ import {
   Text,
   AttributeListStart,
   Identifier,
+  BlockCommentEnd,
 } from "./tokens";
 import { tokenCategoryFilter } from "./tokenCategoryFilter";
 
-// After a command start tag, there may be an attributes list or ID until the
-// end of line.
-const CommandAttributesMode = [
-  AttributeListStart,
-  Identifier,
-  Space,
-  { ...Newline, POP_MODE: true },
-];
 
 // Generates a Lexer for the given language comment patterns.
 export function makeLexer(languageTokens: TokenType[]): Lexer {
   const modes = {
     RootMode: makeRootMode(languageTokens),
-    CommandAttributesMode,
+    // After a command start tag, there may be an attributes list or ID until the
+   // end of line or a block comment end.
+    CommandAttributesMode: [
```

Comment:
Why move this down?

Reply:
I needed access to the `BlockCommendEnd` token as I needed it to be parseable from the `CommandAttributesMode` state if we want to support block comments being able to be defined on a single line. I wanted access to the `languageTokens` array, so I moved this into the `makeLexer` function. Let me know if you disagree with thinking!